src/RazorpayClient.cs

```diff
@@ -26,6 +27,7 @@ public RazorpayClient(string key, string secret)
         {
             RazorpayClient.Key = key;
             RazorpayClient.Secret = secret;
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;

```

Comment:
Issue with this: https://razorpay.slack.com/archives/C2ZL6H76U/p1563980360020500?thread_ts=1563980337.020200&cid=C2ZL6H76U

Reply:
System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;